src/cli/log.rs

```diff
@@ -1,94 +1,149 @@
 use std::{fmt, io::Write};
 
+use termcolor::{Color, ColorSpec, WriteColor};
+use tracing::{level_filters::LevelFilter, Event, Subscriber};
+use tracing_subscriber::{
+    fmt::{
+        format::{self, FormatEvent, FormatFields},
+        FmtContext,
+    },
+    registry::LookupSpan,
+    EnvFilter, Layer,
+};
+
 #[cfg(feature = "otel")]
 use once_cell::sync::Lazy;
 #[cfg(feature = "otel")]
 use opentelemetry_sdk::trace::Tracer;
-#[cfg(feature = "otel")]
-use tracing::Subscriber;
-#[cfg(feature = "otel")]
-use tracing_subscriber::{registry::LookupSpan, EnvFilter, Layer};
 
-use crate::currentprocess::{process, terminalsource};
+use crate::{currentprocess::Process, utils::notify::NotificationLevel};
 
-macro_rules! warn {
-    ( $ ( $ arg : tt ) * ) => ( $crate::cli::log::warn_fmt ( format_args ! ( $ ( $ arg ) * ) ) )
+macro_rules! debug {
```

Comment:
I'd like to keep these in their original order so that the diff more clearly shows what changed exactly.

Reply:
@djc I've split the reordering in another commit. This helps parsing the different levels when I was writing the patch, since `tracing::Level` and `NotificationLevel` have different nomenclatures.